init.lua

```diff
@@ -1,3 +1,6 @@
 local parent, ns = ...
 ns.oUF = {}
 ns.oUF.Private = {}
+
+ns.oUF.isClassic = select(4, GetBuildInfo()) == 11302
+ns.oUF.isRetail = not ns.oUF.isClassic
```

Comment:
I'd stick with just one or the other.

Reply:
It is more conveniece than anything else. Had `isClassic` only first then got annoyed having to type `not isClassic`.